geode-serialization/src/main/java/org/apache/geode/internal/serialization/KnownVersion.java

```diff
@@ -249,7 +249,7 @@
   @Immutable
   public static final KnownVersion GEODE_1_14_0 =
       new KnownVersion("GEODE", "1.14.0", (byte) 1, (byte) 14, (byte) 0, (byte) 0,
-          GEODE_1_14_0_ORDINAL, true);
+          GEODE_1_14_0_ORDINAL);
```

Comment:
On the `support/1.14` branch the `GEODE_1_14_0` `KnownVersion` is constructed with `modifiesClientServerProtocol` set to `true`. Why does this PR change it back to `false`?